### PR TITLE
Add delete method, dry up code a little, update example to use said delete method

### PR DESCRIPTION
**Pull request by SylvainBigonneau**
I think we can add the possibility to delete a picture using a nice `delete` method.

I was not too sure about how to dry up the API settings code (so it would not be redundant in both functions), so I created this anonymous function on top.

I also updated the example in order to use the delete method, but somehow on my machine it does not find my new delete function... you should maybe try it on your config.

### CODE REVIEW

File: example/example.js [hunk 1]
```diff
@@ -10,7 +13,8 @@ if (Meteor.isClient) {
   });
 
   Template.body.events({
-    'click button': function () {
+    'click button#create': function () {
+      $('button').prop('disabled', true);
```
Comment (stubailo) [inline example/example.js]: Please use Session for this instead of setting the property with jQuery?

**Comment by stubailo** [comment]
Looks awesome! Let's just fix up the example a tiny bit, and I'll merge it.

**Comment by BraveKenny** [comment]
There you go!

**Comment by stubailo** [comment]
Published 1.0.3

**Pull request merged**